### ISSUE TITLE
Connection error handling

### ISSUE THREAD

**Issue by KrishnaPG** [numtel/mysql-live-select#2]
Looking at this new interface. One very good point that was missing from your earlier version, as below.

Please bring back the connection error handling. In the new code `db.connect()` is inside [LiveMySql()](https://github.com/numtel/mysql-live-select/blob/master/lib/LiveMysql.js#L27) and does not have much chance for graceful error handling or logging.

For example, please make the `LiveMySql` accept optional error handling callback, (just like your earlier `mysql.createConnection.connect` used to have). That was it would be something like
````
var liveDb = new LiveMysql(Meteor.settings.mysql, function (err) {
    if (err)
        console.error("Error connecting to db:");
    else
        console.log("Connected to DB");
});
````

**Comment by numtel** [comment]
Thank you for this issue. I apologize for the oversight. I have released version 0.0.14 of this package as well as version 0.1.1 of `numtel:mysql` with this feature implemented exactly as you have described.

**Comment by KrishnaPG** [comment]
Thank you much Ben. The new version with callback is very helpful, especially for tracking failures / successes. I am trying to wireup the old one with this new version. Sorry for troubling you. But your work is going to be helpful for many - thanks much for sharing it.
